cyences_app_for_splunk/default/app.conf

```diff
@@ -12,7 +12,7 @@ label = Cyences App for Splunk
 [launcher]
 author = CrossRealms International
 description = Cyences App for Splunk built by CrossRealms International
-version = 4.2.1
+version = 4.3.0
```

Comment:
Just grep to see that's the place we have the version number mentioned, just to ensure.